src/components/Breadcrumbs/Breadcrumbs.cy.tsx

```diff
@@ -20,16 +20,19 @@ const BREADCRUMB_ITEMS_MIXED_ELEMENTS = [
     { label: 'item with link', link: '/some-third-link' },
 ];
 
-const ChangingBreadcrumbs: FC<BreadcrumbsProps> = () => {
-    const [items, setItems] = useState(BREADCRUMB_ITEMS);
+const ChangingBreadcrumbs = ({ items }: BreadcrumbsProps): ReactElement => {
```

Comment:
Why the change?

Reply:
Needed to correctly type the function and now needed to replace the already used prop `items`, which was in use but also passed BREADCRUM_ITEMS 😸